pkg/transform/snowplow_gtmss_preview_test.go

```diff
@@ -281,13 +315,21 @@ func TestExtractHeaderValue(t *testing.T) {
 			Error:    nil,
 		},
 		{
-			Scenario: "invalid_header_value",
+			Scenario: "invalid_header_value (no string)",
 			Event:    fakeSpTsvParsed,
 			Ctx:      "contexts_com_snowplowanalytics_snowplow_web_page_1",
 			Prop:     "id",
 			Expected: nil,
 			Error:    errors.New("invalid header value"),
```

Comment:
If the header value is NOT provided, we don't want to fail it - we want to proceed without attaching the header. Is that the behaviour we have?

Reply:
We check if header value is not null https://github.com/snowplow/snowbridge/blob/1ddd78121cfeac7a54205ceb967d9433d7b7dabf/pkg/transform/snowplow_gtmss_preview.go#L68 and proceed normally if not present.

I think in this test case it should be more `not a string`, so it exists but with different type

Reply:
Super, thanks for confirming!